plugins/inputs/win_eventlog/util.go

```diff
@@ -0,0 +1,155 @@
+//+build windows
+
+//revive:disable-next-line:var-naming
+// Package win_eventlog Input plugin to collect Windows Event Log messages
+package win_eventlog
+
+import (
+	"bytes"
+	"encoding/xml"
+	"fmt"
+	"io"
+	"strings"
+	"unicode/utf16"
+	"unicode/utf8"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+// DecodeUTF16 to UTF8 bytes
+func DecodeUTF16(b []byte) ([]byte, error) {
+
+	if len(b)%2 != 0 {
+		return nil, fmt.Errorf("Must have even length byte slice")
```

Comment:
Error strings should not be capitalized: https://github.com/golang/go/wiki/CodeReviewComments#error-strings